armi/reactor/grids/grid.py

```diff
@@ -39,6 +39,19 @@ class Grid(ABC):
         :id: I_ARMI_GRID_NEST
         :implements: R_ARMI_GRID_NEST
 
+        The reactor will usually have (i,j,k) coordinates to define a
+        simple mesh for locating objects in the reactor. But inside that mesh can
+        be a smaller mesh to define the layout of pins in a reactor, or fuel pellets in
+        a pin, or the layout of some intricate ex-core structure.
+
+        Every time the :py:class:` armi.reactor.grids.locations.IndexLocation` of an
```

Comment:
is this space between ` and armi going to render OK?

Reply:
No idea, so I fixed it.